tests/templates/loops.html

```diff
@@ -11,8 +11,8 @@
       {% if number_reviews > 10 or show_more %}
         <p>All reviews</p>
         {% for review in reviews %}
-          <h3>{{review.title}}</h3>
-          {% for paragraph in review.paragraphs %}
+          <h3>{{review.title}}</h3>{% set rev = review %}
```

Comment:
I was trying to make a minimal change. Overall, yes this needs to be a separate test. And `lookup_variable` needs some refactoring now. But I'm not sure I'll have more time to spent on this.